packages/myst-transforms/src/code.ts

```diff
@@ -13,6 +13,7 @@ type Options = {
 export function codeTransform(mdast: GenericParent, file: VFile, opts?: Options) {
   const code = selectAll('code', mdast) as Code[];
   code.forEach((node) => {
+    if (node.lang === '') return; // raw code cell passes through with no warning
```

Comment:
doing this means that notebook cells of `type=raw` do not cause missing language warnings on the console. This seems ok? unless there is another mechanism? e.g. set lang to `raw` when processing the notebook and test for `raw` here? i'm not sure what the namespace of possible `lang` strings are....?

Reply:
Yeah, I think this is ok... Needing to depend on an empty string is slightly magic - I think `text` would also be a sensible value for `lang`... but I don't know if that has different implications than a raw cell.

Reply:
Just landing this as-is - Hopefully the comment is enough to remind us why we have this check.